app/models/miq_schedule_worker/jobs.rb

```diff
@@ -24,10 +24,6 @@ def miq_server_queue_update_registration_status
     queue_work(:class_name  => "MiqServer", :method_name => "queue_update_registration_status", :server_guid => MiqServer.my_guid)
   end
 
-  def miq_server_resync_rhn_mirror
-    queue_work(:class_name  => "MiqServer", :instance_id => MiqServer.my_server.id, :method_name => "resync_rhn_mirror", :server_guid => MiqServer.my_guid, :msg_timeout => 60.minutes, :task_id => "resync_rhn_mirror")
```

Comment:
In my deletes, I realized these messages might be on the queue, so you might need a data migration to delete them?